pytest_operator/plugin.py

```diff
@@ -329,18 +329,20 @@ async def build_charm(self, charm_path):
         if layer_path.exists():
             # Handle older, reactive framework charms.
             check_deps("charm")
-            cmd = ["charm", "build", "-F", charm_abs]
+            cmd = ["charm", "build", "-F"]
         else:
             # Handle newer, operator framework charms.
-            cmd = ["sg", "lxd", "-c", f"charmcraft build -f {charm_abs}"]
+            cmd = ["sg", "lxd", "-c", "charmcraft pack"]
```

Comment:
`charmcraft pack` (without `-p`) has odd behaviour around copying files via `charmcraft.yaml`.  Wildcards do not work like expected.  Including `-p path` somehow fixes things...
```suggestion
            cmd = ["sg", "lxd", "-c", f"charmcraft pack -p {charm_abs}"]
```

See canonical/charmcraft#514 for more.

Reply:
Removed the workaround for this since it didn't help.